packages/spectral/src/serverTypes/convertIntegration.ts

```diff
@@ -48,28 +48,35 @@ export const convertIntegration = (
   // inline as part of the integration definition.
   const referenceKey = uuid();
 
-  const configVars: Record<string, ConfigVar> = Object.values(
-    definition.configPages ?? {}
-  ).reduce<Record<string, ConfigVar>>(
-    (acc, configPage) =>
-      Object.entries(configPage.elements).reduce<Record<string, ConfigVar>>(
-        (acc, [key, element]) => {
-          // "string" elements are HTML elements and should be ignored.
-          if (typeof element === "string") {
-            return acc;
-          }
-
-          if (key in acc) {
-            throw new Error('Duplicate config var key "' + key + '"');
-          }
-
-          return {
-            ...acc,
-            [key]: element,
-          };
-        },
-        acc
+  const configVars: Record<string, ConfigVar> = Object.values({
+    configPages: definition.configPages ?? {},
+    userLevelConfigPages: definition.userLevelConfigPages ?? {},
+  }).reduce<Record<string, ConfigVar>>(
+    (acc, configPages) => ({
+      ...acc,
+      ...Object.values(configPages).reduce<Record<string, ConfigVar>>(
+        (acc, configPage) =>
+          Object.entries(configPage.elements).reduce<Record<string, ConfigVar>>(
+            (acc, [key, element]) => {
+              // "string" elements are HTML elements and should be ignored.
+              if (typeof element === "string") {
+                return acc;
+              }
+
+              if (key in acc) {
+                throw new Error('Duplicate config var key "' + key + '"');
```

Comment:
```suggestion
                throw new Error(`Duplicate config var key ${key}`);
``` 
This might be easier to read